applications/StructuralMechanicsApplication/tests/cpp_tests/test_truss.cpp

```diff
@@ -32,6 +32,8 @@ class StubBilinearLaw : public ConstitutiveLaw
         mStrain{Strain}, mTangentModuli{TangentModulus1, TangentModulus2}
     {}
 
+    StubBilinearLaw() = default;
+
```

Comment:
```suggestion
```

better let the compiler do its job

Reply:
Thank you for the suggestion, I agree we shouldn't define default constructors if it's not necessary. However, since we have specified a user-declared constructor in line 31, the default constructor won't be generated automatically (see also https://en.cppreference.com/w/cpp/language/default_constructor).

I double checked and indeed when removing it, I got the following error:
```
error C2512: '`anonymous-namespace'::StubBilinearLaw::StubBilinearLaw': no appropriate default constructor available
```